build-normaliz.sh

```diff
@@ -116,3 +116,12 @@ else
 fi
 make -j4
 make install
+
+if [ "$(expr substr $(uname -s) 1 6)" == "CYGWIN" ]; then
```

Comment:
Very minor point: I am not sure that is POSIX portable? The script should not break here if run in e.g. dash or zsh. I can suggest a different check later (when I am at my laptop).

Reply:
This one ought to work in all POSIX compatible shells:
```suggestion
osname=$(uname -s)
if [ "${osname#*$CYGWIN}" != "$osname" ]; then
```